index.d.ts

```diff
@@ -1643,6 +1643,11 @@ declare module binaryen {
     addTable(name: string, initial: number, maximum: number, type: Type): TableRef;
```

Comment:
```suggestion
    addTable(name: string, initial: number, maximum: number): TableRef;
```

`getElementSegmentInfo` seems to be missing by the way.

Reply:
it's here: [pointer](https://github.com/thejchap/binaryen.js/blob/b42dc6cf91d681b8a07334e24870d8a0786ae110/index.d.ts#L2078)

Reply:
Oh, you're right, thanks.